src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -49,223 +49,1826 @@
+					var alwaysPlayCommercials = commercials.alwaysPlayCommercials;
+
+					//Set
+					var commercials = $(".selector").igVideoPlayer("option", "commercials");
+					commercials.alwaysPlayCommercials = true;
+				```
+				*/
 				alwaysPlayCommercials: false,
-				/* type="bool" Get or set whether to show commercial locations or not. */
+				/* type="bool" Get or set whether to show commercial locations or not.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+						commercials: {
+							showBookmarks: false
+						}
+					});
+
+					//Get
+					var commercials = $(".selector").igVideoPlayer("option", "commercials");
+					var showBookmarks = commercials.linkedCommercials.showBookmarks;
+
+					//Set
+					var commercials = $(".selector").igVideoPlayer("option", "commercials");
+					commercials.linkedCommercials.showBookmarks = false;
+				```
+				*/
 				showBookmarks: true,
-				/* type="object" Customizes the ad message settings of the commercial. Ad message shows the duration of the commercial and pops up when the commercial starts playing.  */
+				/* type="object" Customizes the ad message settings of the commercial. Ad message shows the duration of the commercial and pops up when the commercial starts playing.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+						commercials: {
+							linkedCommercials: [
+							{
+								sources: [  "videos/QuinceIntro_1.h264.mp4",
+											"videos/QuinceIntro_1.webmvp8.webm",
+											"videos/QuinceIntro_1.theora.ogv" ],
+								startTime: 20,
+								title: "Quince Presentation p1",
+								link: "http://quince.infragistics.com/"
+							},
+							{
+								sources: [  "videos/QuinceIntro_Part3_1.h264.mp4",
+											"videos/QuinceIntro_Part3_1.webmvp8.webm",
+											"videos/QuinceIntro_Part3_1.theora.ogv" ],
+								startTime: 100,
+								title: "Quince Presentation p3",
+								link: "http://quince.infragistics.com/"
+							}],
+							adMessage: {
+								animate: true,
+								animationDuration: 1000,
+								autoHide: true,
+								hideDelay: 5000
+							}
+						}
+					});
+
+					//Get
+					var commercials = $(".selector").igVideoPlayer("option", "commercials");
+					var adMessage = commercials.adMessage;
+
+					//Set
+					var commercials = $(".selector").igVideoPlayer("option", "commercials");
+					var adMessage = {
+								animate: true,
+								animationDuration: 1000,
+								autoHide: true,
+								hideDelay: 5000
+							};
+					commercials.adMessage = adMessage;
+				```
+				*/
 				adMessage: {
-					/* type="bool" Get or set whether to apply an animation effect when showing or hiding the ad message. If set to true, the animation is played for commercial.adMessage.animationDuration in milliseconds. */
+					/* type="bool" Get or set whether to apply an animation effect when showing or hiding the ad message. If set to true, the animation is played for commercial.adMessage.animationDuration in milliseconds.
```

Comment:
animationDuration should be an option link.

`[animationDuration](ui.igvideoplayer#options:commercials.adMessage.animationDuration)`